api/proto/v1/metrics.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
+
+package api.proto.v1;
+
+import "buf/validate/validate.proto";
+
+option go_package = "github.com/a-x-a/go-metric/api/proto/v1";
+
+// Metric тип описывающий метрику.
+message Metric {
+  required string id = 1 [(validate.field).string.min_len = 1]; // имя метрики
```

Comment:
Please pay attention to that warning
```
field api.proto.v1.Metric.id: label 'required' is not allowed in proto3 or editions
```
https://stackoverflow.com/questions/31801257/why-required-and-optional-is-removed-in-protocol-buffers-3